src/site/content/en/learn/forms/index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Learn Forms
+authors:
+  - andybell
+description: An evergreen CSS course and reference to level up your web styling expertise.
+date: 2021-05-04
+tags:
+  - css
+  - layout
+---
+
+## Welcome to Learn CSS!
+
+This course breaks down the fundamentals of CSS into digestible,
+easy to understand pieces.
+Over the next few modules,
+you'll learn how the core aspects of CSS work and how to use them effectively in your projects. Use the menu pane by the "Learn CSS" logo to navigate the modules.
+
+You'll learn CSS fundamentals like the box model, cascade and specificity, flexbox, grid and z-index.
+And, along with these fundamentals,
+you'll learn about functions, color types, gradients, logical properties and inheritance
+to make you a well-rounded front-end developer,
+ready to take on any user interface.
+
+Each module is full of interactive demos and self-assessments for you to test your knowledge.
+In addition to learning through reading and demos,
+there is an accompanying podcast episode for each topic as another way to learn and continue expanding your knowledge.
+
+This course is created for beginner and advanced CSS developers alike.
+You can go through the series from start to finish
+to get a general understanding of CSS from top to bottom,
+or you can use it as a reference for specific styling subjects.
+For those new to web development overall, check out the
+[intro to HTML](https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML)
+course from MDN to learn all about how to write markup and link your stylesheets.
+
+Here's what you'll learn:
+
+{% include 'partials/course-index.njk' %}
+
+So, are you ready to learn CSS? [Let's get started](/learn/css/box-model/).
+
+{% Aside %}
+This course is written and edited by CSS community experts: <a href="https://twitter.com/piccalilli_">Andy Bell</a>, <a href="https://twitter.com/rachelandrew">Rachel Andrew</a>, <a href="https://twitter.com/Una">Una Kravets</a>, and <a href="https://twitter.com/argyleink">Adam Argyle</a>. <a href="https://twitter.com/rob_dodson">Rob Dodson</a> and <a href="https://twitter.com/jiwoong">Jiwoong Lee</a> shepherded the UX & production build, along with help from <a href="https://twitter.com/devnook">Ewa Gasperowicz</a> and <a href="https://twitter.com/piccalilli_">Kayce Basques</a>.
```

Comment:
nit: should this go to web.dev authors pages eventually? Or why is this here vs. all being listed in the front matter? @andy-piccalilli

Reply:
If I remember rightly this got slotted into the CSS course post-launch. There's a credits section right at the end of the course too.

Reply:
oh I didn't notice this aside was on the overview page.

To answer your question Sam, because there are so many authors, we put them at the end of the course:
https://web.dev/learn/css/conclusion/#brought-to-you-by

I did this because otherwise we'd end up with a huge stack of authors on each page that would force down the first heading and look a bit odd.

Reply:
Sure. I'm a bit confused because what you've linked to @robdodson feels structured and this part clearly is not. My point is more—if this is important it should be a front-matter list of contributors, which I feel like what was linked to definitely is.

Reply:
yeah, I'm not sure how this Aside was added. The thing I wrote uses the authors in the frontmatter and sends them to a macro. https://github.com/GoogleChrome/web.dev/blame/main/src/site/content/en/learn/css/conclusion/index.md

I can delete the aside from this PR so it doesn't encourage the Learn Forms authors to do the same thing.